dateparser/date.py

```diff
@@ -175,7 +175,7 @@ def _try_given_formats(self):
         if not self.date_formats:
             return
 
-        return parse_with_formats(self._get_translated_date_with_formatting(), self.date_formats)
+        return parse_with_formats(self._get_translated_date_with_formatting(), list(self.date_formats))
```

Comment:
Right, I never got around to running a test, should have instead put it in
a list if it's a string. Haven't looked at this in a while

On Mon, Feb 9, 2015, 5:39 PM Eugene Amirov <notifications@github.com> wrote:

> In dateparser/date.py
> <https://github.com/scrapinghub/dateparser/pull/18#discussion_r24372801>:
>
> > @@ -175,7 +175,7 @@ def _try_given_formats(self):
> >          if not self.date_formats:
> >              return
> >
> > -        return parse_with_formats(self._get_translated_date_with_formatting(), self.date_formats)
> > +        return parse_with_formats(self._get_translated_date_with_formatting(), list(self.date_formats))
>
> But how would this change help in such case?
>
> >>> list("qwerty")
> ['q', 'w', 'e', 'r', 't', 'y']
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scrapinghub/dateparser/pull/18/files#r24372801>.
>